CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 0.7.1 (2020-01-29)
+
+We have accumulated enough neat features to be worthy of a fresh release. Enjoy!
```

Comment:
Actually, a feature release would need to be 0.8.0, which this is not suitable for. I'd announce it as a badly needed fix (#182 is one) which shipped some features as well